### PR TITLE
Update CONTRIBUTING with easyjson

### LINKED ISSUE #90: Document how to update EasyJson file in CONTRIBUTING.md
#88 introduced [EasyJson](https://github.com/mailru/easyjson) to this repo. It autogenerates a file `types_easyjson.go` by using the command 

```bash
easyjson -all types.go
```

This should be documented in the CONTRIBUTING.md

More info at https://github.com/mailru/easyjson#usage

### PR DESCRIPTION
**Pull request by eskombro**
Closes #90

### CODE REVIEW

**Review by senyast4745 — APPROVED** [review]
All is good

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,6 +54,19 @@ $ gofmt -w ./..
 
 > To find out where golint was installed you can run `go list -f {{.Target}} golang.org/x/lint/golint`. For golint to be used globally add that directory to the $PATH environment setting (taken from golint docs).
 
+### EasyJson
+
+[`easyjson`](https://github.com/mailru/easyjson) is a package used for optimizing marshal/unmarshal Go structs to/from JSON. It takes the `types.go` file as an input, and auto-generates `types_easyjson.go` with optimized marshalling and unmarshalling methods for this SDK.
+
+If for any reason `types.go` is modified, this file should be regenerated by running easyjson again.
+
+#### install easyjson
```
Comment (curquiza) [inline CONTRIBUTING.md:63]:
```suggestion
#### Install easyjson

```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,6 +54,19 @@ $ gofmt -w ./..
...
+go get -u github.com/mailru/easyjson/...
+
+#### regenerate types_easyjson.go
```
Comment (curquiza) [inline CONTRIBUTING.md:66]:
```suggestion
#### Regenerate `types_easyjson.go`
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -54,6 +54,19 @@ $ gofmt -w ./..
...
+easyjson -all types.go
+
+
```
Comment (curquiza) [inline CONTRIBUTING.md:69]:
```suggestion
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -54,6 +54,19 @@ $ gofmt -w ./..
...
 > To find out where golint was installed you can run `go list -f {{.Target}} golang.org/x/lint/golint`. For golint to be used globally add that directory to the $PATH environment setting (taken from golint docs).
 
+### EasyJson
+
+[`easyjson`](https://github.com/mailru/easyjson) is a package used for optimizing marshal/unmarshal Go structs to/from JSON. It takes the `types.go` file as an input, and auto-generates `types_easyjson.go` with optimized marshalling and unmarshalling methods for this SDK.
+
+If for any reason `types.go` is modified, this file should be regenerated by running easyjson again.
+
+#### install easyjson
+go get -u github.com/mailru/easyjson/...
```
Comment (curquiza) [inline CONTRIBUTING.md:64]: Could you add this line in a "```bash" please? 🙂

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -54,6 +54,19 @@ $ gofmt -w ./..
...
+
+#### regenerate types_easyjson.go
+easyjson -all types.go
```
Comment (curquiza) [inline CONTRIBUTING.md:67]: Could you add this line in a "```bash" please? 🙂

**Review by curquiza — CHANGES_REQUESTED** [review]

**Pull request merged**